CONTRIBUTING.md

```diff
@@ -155,6 +155,105 @@ toolkit for interacting with AWS backend resources.
 2. [AWS SDK for iOS](https://github.com/aws-amplify/aws-sdk-ios)
 3. [AWS SDK for JavaScript](https://github.com/aws/aws-sdk-js)
 
+## Integration Tests
```

Comment:
view in MD https://github.com/ragingsquirrel3/amplify-flutter/blob/tshepp/integration_tests_init/CONTRIBUTING.md#integration-tests